<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateMachineDialog</class>
 <widget class="QDialog" name="CreateMachineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>70</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>虚拟机名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFormat">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>130</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>qcow2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>虚拟机格式：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSize">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>190</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>文件大小：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonYes">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>240</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>文件路径：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPath">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
